src/main/sensors/gyroanalyse.c

```diff
@@ -49,25 +49,23 @@
 // for gyro loop >= 4KHz, sample rate 2000 defines FFT range to 1000Hz, 16 bins each 62.5 Hz wide
 // NB  FFT_WINDOW_SIZE is set to 32 in gyroanalyse.h
 #define FFT_BIN_COUNT             (FFT_WINDOW_SIZE / 2)
-// start to compare 3rd bin to 2nd bin, ie start comparing from 77Hz, 100Hz, or 150Hz centres
-#define FFT_BIN_OFFSET            2
 // smoothing frequency for FFT centre frequency
 #define DYN_NOTCH_SMOOTH_FREQ_HZ  50
 // we need 4 steps for each axis
 #define DYN_NOTCH_CALC_TICKS      (XYZ_AXIS_COUNT * 4)
 
 #define DYN_NOTCH_OSD_MIN_THROTTLE 20
 
-static uint16_t FAST_RAM_ZERO_INIT   fftSamplingRateHz;
-static float FAST_RAM_ZERO_INIT      fftResolution;
-static uint8_t FAST_RAM_ZERO_INIT    fftBinOffset;
-static uint16_t FAST_RAM_ZERO_INIT   dynNotchMaxCtrHz;
-static uint8_t dynamicFilterRange;
-static float FAST_RAM_ZERO_INIT      dynNotchQ;
-static float FAST_RAM_ZERO_INIT      dynNotch1Ctr;
-static float FAST_RAM_ZERO_INIT      dynNotch2Ctr;
-static uint16_t FAST_RAM_ZERO_INIT   dynNotchMinHz;
-static bool FAST_RAM dualNotch = true;
+static uint16_t FAST_RAM_ZERO_INIT fftSamplingRateHz;
+static float    FAST_RAM_ZERO_INIT fftResolution;
+static uint8_t  FAST_RAM_ZERO_INIT fftStartBin;
+static uint16_t FAST_RAM_ZERO_INIT dynNotchMaxCtrHz;
+static uint8_t                     dynamicFilterRange;
+static float    FAST_RAM_ZERO_INIT dynNotchQ;
+static float    FAST_RAM_ZERO_INIT dynNotch1Ctr;
+static float    FAST_RAM_ZERO_INIT dynNotch2Ctr;
+static uint16_t FAST_RAM_ZERO_INIT dynNotchMinHz;
+static bool     FAST_RAM           dualNotch = true;
```

Comment:
Please do not align stuff - as this shows, this will only create a lot of churn, making it hard to spot actual change in reviews amongst the formatting changes required to preserve the alignment. :-(

Reply:
Understood.